topydo/lib/PrettyPrinterFilter.py

```diff
@@ -57,15 +60,21 @@ def filter(self, p_todo_str, p_todo):
             except KeyError:
                 pass
 
-            p_todo_str = color + p_todo_str + NEUTRAL_COLOR
-
+            p_todo_str = '%s%s%s' % (color, p_todo_str, NEUTRAL_COLOR)
```

Comment:
> I believe the second form is also Python 3.x compliant.

Concatenating strings is still valid operation on python 3.x Consider https://docs.python.org/3/library/stdtypes.html#common-sequence-operations

So it's up to @bram85 to decide which option to choose.